### DISCUSSION TITLE
AlDente Charging Limit

### DISCUSSION THREAD

**Discussion by omergunek** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#633] [Q&A] [answered]
I have been using AlDente for a long time. My battery health is 97%, the cycle is 11. I think I'm setting the wrong charge limit because my battery health is declining fast. What do you think my charge limit should be?

**Comment by MatthiasKerbl** [comment, marked as answer]
> I have been using AlDente for a long time. My battery health is 97%, the cycle is 11. I think I'm setting the wrong charge limit because my battery health is declining fast. What do you think my charge limit should be?

Hi @omergunek,

Your battery will lose 5-10% capacity in the first year. That is completely normal. What matters is what happens after the first 5-10% capacity loss. With AlDente and the correct behavior, you can slow down the aging of your battery.
When it comes to the question of "What is the perfect charge limit for best long-term battery health?", we just do not know exactly or at least I do not know. Different research papers show different results reaching from 40%-60%. However, when you have your SoC limited to 40-60% you will miscalibrate your battery more quickly. Therefore, you would need to calibrate your battery more often. In my opinion, 80% might be a good middle-ground for a lot of people but you have to decide on your own.

Cheers,
Matthias
